arch/risc-v/src/common/riscv_exception_common.S

```diff
@@ -28,8 +28,12 @@
 #include <arch/irq.h>
 #include <arch/mode.h>
 
+#include <sys/types.h>
```

Comment:
why need

Reply:
But this macro isn't defined in sys/types.h

Reply:
From sys/types.h, lines 63-65

```
#ifndef CONFIG_SMP_NCPUS
#  define CONFIG_SMP_NCPUS 1
#endif
```
I'm not saying I fully understand why this is here and not in config.h, but it is there now

Reply:
Ok, understand. It's because CONFIG_SMP_NCPUS isn't defined if CONFIG_SMP equal n.